ts/src/bitget.ts

```diff
@@ -3451,7 +3451,8 @@ export default class bitget extends Exchange {
                     }
                 }
             }
-            const orderRequest = this.createOrderRequest (marketId, type, side, amount, price, orderParams);
+            let orderRequest = this.createOrderRequest (marketId, type, side, amount, price, orderParams);
+            orderRequest = this.omit (orderRequest, 'marginMode');
```

Comment:
@Dan-krm this is not needed I think, createOrderRequest already omits `marginMode` through the `[ marginMode, params ] = this.handleMarginModeAndParams ('createOrder', params);` call